colors/oxocarbon-dark.lua

```diff
@@ -56,6 +55,17 @@ style.divider = base00 -- borders
 style.line_number = base03 -- inactive
 style.line_number2 = base04 -- active
 
+style.drag_overlay = { common.color "rgba(255, 255, 255, 0.30)" }
+style.drag_overlay_tab = base03
+
+style.scrollbar = base01 -- scrollbar button when not interacted
+style.scrollbar2 = base03 -- button when you hover over
+style.scrollbar_track = base01 -- scrollbar track
+
+style.nagbar = base10
+style.nagbar_text = base01
+style.nagbar_dim = { common.color "rgba(0, 0, 0, 0)" }
```

Comment:
Why fully transparent?

Reply:
Felt like it looked right, but now that you mentioned it, it is probably a good idea to atleast make it darken the screen a little bit